src/vs/workbench/browser/parts/editor/editorGroupsControl.ts

```diff
@@ -193,6 +214,10 @@ export class EditorGroupsControl extends Themable implements IEditorGroupsContro
 		return this.layoutVertically ? EditorGroupsControl.MIN_EDITOR_WIDTH : EditorGroupsControl.MIN_EDITOR_HEIGHT;
 	}
 
+	private get visibleSilos(): number {
```

Comment:
This is not needed, use `this.editorGroupService.getStacksModel().groups.length`

Reply:
done.